pages/linux/efibootbgr.md

```diff
@@ -0,0 +1,24 @@
+# efibootmgr
+
+> Manipulate the UEFI Boot Manager (the Bootoptions).
+> More information: https://linux.die.net/man/8/efibootmgr.
+
+- List the current settings / bootnums:
+
+`efibootmgr`
+
+- List the Filepaths:
```

Comment:
Should Filepaths be capitalized ? I can make the change myself, but just wanted to check.

Reply:
As a German, I capitalize a lot of words, that the english language doesn't capitalize. 🙃

Thinking about it, I shouldn't be capitalized

Reply:
I think the suggestion made by @agnivade is right and due it was a little change and @sp1ritCS seems to be agree, I performed the change.